stan/math/prim/fun/grad_2F1.hpp

```diff
@@ -36,12 +36,14 @@ namespace math {
  */
 template <typename T>
 void grad_2F1(T& g_a1, T& g_b1, const T& a1, const T& a2, const T& b1,
```

Comment:
I couldn't add a comment above, but the docs are wrong I think:

```
* @param[out] g_a1 g_a1 pointer to array of six values of type T, result.
* @param[out] g_b1 g_b1 pointer to array of six values of type T, result.
```

I think should be

```
* @param[out] g_a1 gradient of hypergeometric function with respect to a1, result
* @param[out] g_b1 gradient of hypergeometric function with respect to b1, result
```

but at least they definitely aren't pointers so should just be:

```
* @param[out] g_a1 result
* @param[out] g_b1 result
```

If we're not sure what they are (the erroneous line appears to be copy pasted from the docs of `grad_F32`: `* @param[out] g g pointer to array of six values of type T, result.`)

Reply:
Thanks for pointing that out, I didn't touch those this time but I'll update it.